meta.go

```diff
@@ -0,0 +1,64 @@
+package spotify
+
+import (
+	"net/url"
+	"strings"
+)
+
+type HREF string
+
+// Get returns the most recent version of the object in it's entirety
+func (h *HREF) Get(api *API, obj interface{}) error {
+	url, err := h.GetURL()
+	if err != nil {
+		return err
+	}
+	path := strings.Replace(url.Path, "/v1", "", 1)
+	return api.get(path, obj)
+}
+
+// GetURL parses HREF into a *net/url.URL
+func (h *HREF) GetURL() (*url.URL, error) {
```

Comment:
I think this is more idiomatic
```suggestion
func (h *HREF) URL() (*url.URL, error) {
```